tests/integration/api_v2/http/delegates.test.js

```diff
@@ -35,7 +35,12 @@ const endpoint = `${baseUrlV2}/accounts`;
 describe('Delegates API', () => {
 	let refDelegate;
 	beforeAll(async () => {
-		[refDelegate] = (await api.get(`${endpoint}?isDelegate=true&limit=1`)).data;
+		let response;
+		do {
+			// eslint-disable-next-line no-await-in-loop
+			response = await api.get(`${baseUrlV2}/accounts?isDelegate=true&limit=1`);
+		} while (!response);
```

Comment:
`response` will always be returned but `response.data` will only ensure that a successful response is received.

```suggestion
		} while (!response.data);
```